src/main/scala/com/databricks/spark/redshift/package.scala

```diff
@@ -76,12 +76,19 @@ package object redshift {
   implicit class RedshiftDataFrame(dataFrame: DataFrame) {
 
     /**
-     * Load the DataFrame into a Redshift database table
+     * Load the DataFrame into a Redshift database table. By default, this will append to the
+     * specified table. If the `overwrite` parameter is set to `true` then this will drop the
+     * existing table and re-create it with the contents of this DataFrame.
      */
     @deprecated("Use DataFrame.write()", "0.5.0")
     def saveAsRedshiftTable(parameters: Map[String, String]): Unit = {
       val params = Parameters.mergeParameters(parameters)
-      DefaultRedshiftWriter.saveToRedshift(dataFrame.sqlContext, dataFrame, params)
+      val saveMode = if (params.overwrite) {
```

Comment:
Here, and reviewers should confirm this, the old non-overwrite behavior was to append, since we'd just end up performing a copy into the existing table.